OpenRobertaServer/staticResources/js/app/roberta/controller/menu.controller.js

```diff
@@ -176,9 +176,19 @@ define([ 'exports', 'log', 'util', 'message', 'comm', 'robot.controller', 'socke
             return groupsDict[key];
         }
 
+        /**
+         * Initialize the navigation bar in the head of the page
```

Comment:
Comment is misleading, this method either changes or removes the info link to the buttons in the carousel/group member view.